.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "external/metal"]
-	path = external/metal
-	url = https://github.com/brunocodutra/metal
```

Comment:
!!!

@trws , you're a deeply alarming person, you know that? I mean that as the highest form of compliment. Seriously nice though